### DISCUSSION TITLE
Changing reward function value for roundabout environment

### DISCUSSION THREAD

**Discussion by JuulLinders** [eleurent/highway-env#308] [Q&A]
Dear Edouard,

I am having trouble editing the reward values for the roundabout-v0 environment.
If I use env.config["collision_reward"] = - 10 it does not seem to use the edited value in calculating the rewards.
Do you have an alternative method to change these values for the roundabout environment?

Kind regards,

Juul Linders

**Comment by eleurent** [comment]
I think it does changes the reward function, but it is always rescaled to the [0, 1] range.
So e.g. if three states for collision, driving slow and driving fast had rewards of -5, 0, 1, these were remapped to 0, 0.83, 1.
If you change the collision penalty to -10 instead, now the three rescaled rewards are 0, 0.91, 1.

I will add a config to allow to disable the normalisation.

**Comment by eleurent** [comment]
Update: if you install the development version, you can now disable the reward normalisation by using
```
env.configure({'normalize_reward': False, 'collision_reward': -10})
```

**Reply by zhangjianguo-max** [reply]
Hello, I installed the stable version, how should I disable the reward normalization with the command

**Reply by eleurent** [reply]
There is a `normalize_reward` config that you can set, but it is not used in all environments. 
Alternatively, you can clone the dev version and edit the reward function directly.
